### PR TITLE
New package: forgejo-1.18.0

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/forgejo/files/forgejo/run [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+export USER=_forgejo
+export HOME=/var/lib/forgejo
+
+export GITEA_WORK_DIR="${HOME}"
```
Comment (paper42) [inline srcpkgs/forgejo/files/forgejo/run:7]: is it really still called GITEA_WORK_DIR?

Comment (tranzystorek-io) [reply srcpkgs/forgejo/files/forgejo/run:7]: afaik yes, many things were left unchanged to enable easier migration from gitea

**Comment by tranzystorek-io** [comment]
Closing as forgejo is too close to be a gitea clone to warrant packaging

File: srcpkgs/forgejo/files/forgejo/run (hunk 1 above)
Comment (return42) [reply srcpkgs/forgejo/files/forgejo/run:7]: > is it really still called GITEA_WORK_DIR?

no, it is: `$FORGEJO_WORK_DIR`

- https://forgejo.org/docs/latest/admin/config-cheat-sheet/#default-configuration-non-appini-configuration

**Comment by return42** [comment]
> Closing as forgejo is too close to be a gitea clone to warrant packaging

This is no longer true / however, it was never true ;-)  .. the reasons why developers prefer forgejo over gitea might related to the motivation: [We started Forgejo in reaction to control of Gitea being taken away from the community by the newly-formed for-profit company Gitea Ltd without prior community consultation](https://forgejo.org/2022-12-15-hello-forgejo/)

@tranzystorekk would you like to reopen this PR? .. BTW: [v8.0.3 (stable)](https://forgejo.org/releases/) has been released / thanks :+1:

**Pull request closed without merging**